src/bootstrap/datatable.html

```diff
@@ -95,7 +95,7 @@
     <tfoot if.bind="footer" innerhtml.bind="footer"></tfoot>
   </table>
 
-  <div show.bind="pages > 1">
+  <div if.bind="!offlineMode" show.bind="pages > 1">
```

Comment:
Offline support needs some kind of pagination. It will now show all the rows otherwise. (that is why people use datatables 9/10 times)